src/org/usfirst/frc/team2585/robot/Environment.java

```diff
@@ -25,33 +28,31 @@
 	public Environment(Robot robot) {
 		super(robot);
 		input = new XboxInput();
-		wheels = new WheelSystem();
-		shooter = new ShooterSystem();
-		intake = new IntakeSystem();
 		
-		wheels.init(this);
-		shooter.init(this);
-		intake.init(this);
+		systems = new HashMap<String, RobotSystem>();
+		systems.put("wheelSystem", new WheelSystem());
```

Comment:
Make the String that accesses the HashMap a public static final variable of Environment. That way you can access it by doing getSystem(Environment.WHEELSYSTEM) instead of having to lookup the name of the key.